ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -941,6 +941,357 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-aws-sts-advanced
+- as: aws-rosa-sts-advanced-f360-content
+  cron: 9 21 2,9,16,25 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      ADDITIONAL_SECURITY_GROUP: "true"
+      CHANNEL_GROUP: nightly
+    test:
+    - as: test-e8-rhcos4
+      cli: latest
+      commands: |
+        set -x
+        export PROFILE=e8
+        export PRODUCT=rhcos4
+        export component=ocp4-content-ds
+        export ROOT_DIR=$PWD
+        git clone https://github.com/ComplianceAsCode/ocp4e2e.git ocp4e2e
+        pushd ocp4e2e
+        go test -v -timeout 120m github.com/ComplianceAsCode/ocp4e2e -profile="$PROFILE" -product="$PRODUCT" -content-image="$CONTENT_IMAGE"
+      dependencies:
+      - env: CONTENT_IMAGE
```

Comment:
I think we are missing a step here, we also need the prow to build the content image. See[ ](https://github.com/openshift/release/blob/fd7608f514ad81557e1e55d4bccfe9fea3ab43de/ci-operator/config/ComplianceAsCode/content/ComplianceAsCode-content-master__4.13.yaml#L6)